psst-gui/src/ui/artist.rs

```diff
@@ -83,6 +96,39 @@ fn async_related_widget() -> impl Widget<AppState> {
 }
 
 pub fn artist_widget() -> impl Widget<Artist> {
+    let artist_image = artist_cover_widget(theme::grid(21.0));
+    Flex::row()
+        .with_child(artist_image)
+        .with_child(
+            Scroll::new(
+                Label::new(
+                    "Coldplay are a British rock band formed in London in 1997, consisting of vocalist and pianist Chris Martin, lead guitarist Jonny Buckland, bassist Guy Berryman, drummer and percussionist Will Champion, and manager Phil Harvey. They are best known for their live performances, having also impacted popular culture with their artistry, advocacy and achievements. \n The members of the band initially met at University College London, calling themselves Big Fat Noises and changing to Starfish, before settling on the current name. After releasing Safety (1998) independently, Coldplay signed with Parlophone in 1999 and wrote their debut album, Parachutes (2000). It featured breakthrough single \"Yellow\" and received a Brit Award for British Album of the Year and a Grammy Award for Best Alternative Music Album. The group's follow-up, A Rush of Blood to the Head (2002), won the same accolades. X&Y (2005) later saw the completion of what they considered a trilogy, being nominated for Best Rock Album as well. Its successor, Viva la Vida or Death and All His Friends (2008), prevailed in the category. Both albums were the best-selling of their years, topping the charts in over 30 countries. Viva la Vida's title track also became the first British act single to lead the Billboard Hot 100 and UK Singles Chart simultaneously in the 21st century."
```

Comment:
Guessing this was just for show at some point?

Reply:
Yes this just shows in the artist section, just a filler for now!! That was funny be the wiki section. Haven't put much thought to it yet though. Probably should be a different pr.